tox.ini

```diff
@@ -12,7 +12,7 @@ whitelist_externals =
     sed
 deps =
     -rtest-requirements.txt
-    ansible{env:ANSIBLE_VERSION:==2.1.0}
+    ansible{env:ANSIBLE_VERSION:==2.2.0}
```

Comment:
You'll likely need to match this to the current version at the SHA you're using.